src/main/java/com/mariamkatamashvlii/gym/model/User.java

```diff
@@ -0,0 +1,50 @@
+package com.mariamkatamashvlii.gym.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
```

Comment:
Please avoid importing all classes from package. *
Import anly what you need for your class.